.devcontainer/Dockerfile

```diff
@@ -62,6 +62,18 @@ RUN mkdir /tmp/cargo-deny && cd /tmp/cargo-deny && \
     mv cargo-deny-0.11.0-x86_64-unknown-linux-musl/cargo-deny $HOME/bin && \
     cd .. && rm -rf /tmp/cargo-deny
 
+RUN for p in \
+    github.com/uudashr/gopkgs/v2/cmd/gopkgs@latest \
+    github.com/ramya-rao-a/go-outline@latest \
+    github.com/cweill/gotests/gotests@latest \
+    github.com/fatih/gomodifytags@latest \
+    github.com/josharian/impl@latest \
+    github.com/haya14busa/goplay/cmd/goplay@latest \
+    github.com/go-delve/delve/cmd/dlv@latest \
+    github.com/golangci/golangci-lint/cmd/golangci-lint@latest \
+    golang.org/x/tools/gopls@latest \
+    ; do go install "$p" ; done
```

Comment:
It might be slightly better to move this into its own build stage and then just copy in the whole `/go` directory... Will prevent needless rebuilds on other changes